src/ImplicitPackageReference/ImplicitPackageReference.cs

```diff
@@ -46,38 +48,54 @@ public AddImplicitPackageReferences(ILogger log)
 
         public override bool Execute()
         {
-            if(DependenciesToVersionAndPackage.Length == 0)
+            if(ImplicitPackageReferences.Length == 0)
             {
                 log.LogMessage("AddImplicitPackageReferences was not given any packages to version");
                 return true;
             }
 
-            if (File.Exists(AssetsFilePath))
+            if (!File.Exists(AssetsFilePath))
             {
-                bool versionlessPackagesFound = false;
-                StringBuilder namesOfVersionlessPackages = new StringBuilder();
-                JObject assetsFile;
-                try
-                {
-                    assetsFile = JObject.Parse(File.ReadAllText(AssetsFilePath));
-                }
-                catch (Exception e)
-                {
-                    log.LogError("AddImplicitPackageReferences failed, could not parse file " + AssetsFilePath + ". Exception:" + e.Message);
-                    return false;
-                }
+                log.LogError("Project.assets.json path does not exist at: " + AssetsFilePath);
+                return false;
+            }
+            bool versionlessPackagesFound = false;
+            StringBuilder namesOfVersionlessPackages = new StringBuilder();
+            JObject assetsFile;
```

Comment:
Yes, it does.  I use it in other projects.  I'm unsure if it provides all of the editing functionality that we want, and migrating to it would potentially cause issues, but I'm not against it.